frontend/src/scenes/session-recordings/filters/SimpleSessionRecordingsFilters.tsx

```diff
@@ -0,0 +1,190 @@
+import {
+    AnyPropertyFilter,
+    EntityTypes,
+    FilterType,
+    PropertyFilterType,
+    PropertyOperator,
+    RecordingFilters,
+} from '~/types'
+import { useMemo, useState } from 'react'
+import { BindLogic, useActions, useValues } from 'kea'
+import { propertyFilterLogic } from 'lib/components/PropertyFilters/propertyFilterLogic'
+import { TaxonomicPropertyFilter } from 'lib/components/PropertyFilters/components/TaxonomicPropertyFilter'
+import { PropertyFilterLogicProps } from 'lib/components/PropertyFilters/types'
+import { Popover } from 'lib/lemon-ui/Popover/Popover'
+import { teamLogic } from 'scenes/teamLogic'
+import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
+import { LemonButton } from '@posthog/lemon-ui'
+import { PropertyFilters } from 'lib/components/PropertyFilters/PropertyFilters'
+
+export const SimpleSessionRecordingsFilters = ({
+    filters,
+    setFilters,
+    localFilters,
+    setLocalFilters,
+}: {
+    filters: RecordingFilters
+    setFilters: (filters: RecordingFilters) => void
+    localFilters: FilterType
+    setLocalFilters: (localFilters: FilterType) => void
+}): JSX.Element => {
+    const { currentTeam } = useValues(teamLogic)
+
+    const personPropertyOptions = useMemo(() => {
+        const properties = [{ label: 'Country', key: '$geoip_country_name' }]
+        const displayNameProperties = currentTeam?.person_display_name_properties ?? []
```

Comment:
Ah! This doesn't work with the defaults

If you've never chosen one then we show a list of person props in settings even though the "setting" is empty.

Then if you re-order the default list we set it to a long list of options

We should probably take just the top one...
![2023-08-01 18 39 19](https://github.com/PostHog/posthog/assets/984817/efb44ecf-240b-42ba-9531-b844dc6a9a00)

Reply:
Great point. I think we have space for two so going to take the first two (if they exist)